_config.yml

```diff
@@ -120,8 +120,8 @@ repos:
 google_analytics_ua: UA-48605964-19
 
 back_link:
-  url: "https://pages.18f.gov/guides/"
-  text: Read more 18F Guides
+  url: https://guides.18f.gov/guides
```

Comment:
I would just say pages.18f.gov/guides if you want to merge now; that will redirect correctly.